modules/unshorten/file.txt

```diff
@@ -0,0 +1,904 @@
+        _ended: true,
+        res: [Circular],
+        aborted: undefined,
+        timeoutCb: null,
+        upgradeOrConnect: false,
+        parser: null,
+        maxHeadersCount: null,
+        [Symbol(outHeadersKey)]: [Object] },
+     read: [Function],
+     _consuming: true,
+     _idleNext: null,
+     _idlePrev: null,
+     _idleTimeout: -1,
+     [Symbol(asyncId)]: 44,
+     [Symbol(bytesRead)]: 232083 },
+  httpVersionMajor: 1,
+  httpVersionMinor: 1,
+  httpVersion: '1.1',
+  complete: true,
+  headers: 
+   { 'x-ebay-c-request-id': 'ri=cZNGYEte0%2Byn,rci=WMm2%2FB4voYcAJFop',
+     rlogid: 't6ulcpjqcj9%3Fjqpsobtlrbn%284743%3F%3E4%2Busqdrrp%2Bjqp%2Bceb%7C%28dlh-163bc6fc484-0x16c',
+     'x-frame-options': 'SAMEORIGIN',
+     'x-content-type-options': 'nosniff',
+     'x-xss-protection': '1; mode=block',
+     'cache-control': 'private',
+     pragma: 'no-cache',
+     'content-type': 'text/html;charset=UTF-8',
+     'content-language': 'en-US',
+     server: 'ebay server',
+     date: 'Fri, 01 Jun 2018 17:40:31 GMT',
+     'transfer-encoding': 'chunked',
+     connection: 'close, Transfer-Encoding',
+     'set-cookie': 
+      [ 'JSESSIONID=6E945BB36963A1C297F47E3E47190ACC; Path=/; HttpOnly',
+        'ebay=%5Esbf%3D%23%5E;Domain=.ebay.com;Path=/ ',
+        'dp1=bu1p/QEBfX0BAX19AQA**5cf2b88f^bl/US5ed3ec0f^;Domain=.ebay.com;Expires=Sun, 31-May-2020 17:40:31 GMT;Path=/ ',
+        's=CgAD4ACBbEtaPYmM2ZmM0NzkxNjMwYWFkY2Y0NjIwN2E3ZmZkYWI4NWWVtK8Q;Domain=.ebay.com;Path=/; HttpOnly ',
+        'nonsession=CgAAIABxbORIPMTUyNzg3NDgzMXgxMTE5OTE0OTczNjl4MHgyTgDLAAFbEYwXMQDKACBkd4aPYmM2ZmM0NzkxNjMwYWFkY2Y0NjIwN2E3ZmZkYWI4NWWREZsE;Domain=.ebay.com;Expires=Sat, 01-Jun-2019 17:40:31 GMT;Path=/ ' ] },
+  rawHeaders: 
+   [ 'X-EBAY-C-REQUEST-ID',
+     'ri=cZNGYEte0%2Byn,rci=WMm2%2FB4voYcAJFop',
+     'RlogId',
+     't6ulcpjqcj9%3Fjqpsobtlrbn%284743%3F%3E4%2Busqdrrp%2Bjqp%2Bceb%7C%28dlh-163bc6fc484-0x16c',
+     'X-Frame-Options',
+     'SAMEORIGIN',
+     'X-Content-Type-Options',
+     'nosniff',
+     'X-XSS-Protection',
+     '1; mode=block',
+     'Cache-Control',
+     'private',
+     'Pragma',
+     'no-cache',
+     'Content-Type',
+     'text/html;charset=UTF-8',
+     'Content-Language',
+     'en-US',
+     'Server',
+     'ebay server',
+     'Date',
+     'Fri, 01 Jun 2018 17:40:31 GMT',
+     'Transfer-Encoding',
+     'chunked',
+     'Connection',
+     'close',
+     'Connection',
+     'Transfer-Encoding',
+     'Set-Cookie',
+     'JSESSIONID=6E945BB36963A1C297F47E3E47190ACC; Path=/; HttpOnly',
+     'Set-Cookie',
+     'ebay=%5Esbf%3D%23%5E;Domain=.ebay.com;Path=/ ',
+     'Set-Cookie',
+     'dp1=bu1p/QEBfX0BAX19AQA**5cf2b88f^bl/US5ed3ec0f^;Domain=.ebay.com;Expires=Sun, 31-May-2020 17:40:31 GMT;Path=/ ',
+     'Set-Cookie',
+     's=CgAD4ACBbEtaPYmM2ZmM0NzkxNjMwYWFkY2Y0NjIwN2E3ZmZkYWI4NWWVtK8Q;Domain=.ebay.com;Path=/; HttpOnly ',
```

Comment:
Note, you might want to reset your password or logout/login if this is from a logged in session; I'm not checking the cookie to see if it's valid, but either way it's out there now.

Reply:
Thanks for the heads up!